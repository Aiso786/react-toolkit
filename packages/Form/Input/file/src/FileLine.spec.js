import React from 'react';
import { shallow } from 'enzyme';
import FileLine from './FileLine';

describe('<File.FileInput>', () => {
  it('renders File.FileInput correctly', () => {
    const wrapper = shallow(
      <FileLine file={{ name: 'name', size: 1, preview: '#' }} id="id" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
