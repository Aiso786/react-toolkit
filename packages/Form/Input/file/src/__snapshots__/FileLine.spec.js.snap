// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<File.FileInput> renders File.FileInput correctly 1`] = `
<tr
  className="custom-line-file af-file-line"
>
  <td
    className="text-center"
  >
    <Preview
      file={
        Object {
          "name": "name",
          "preview": "#",
          "size": 1,
        }
      }
    />
  </td>
  <td>
    name
  </td>
  <td
    className="text-center"
  >
    1
  </td>
  <td
    className="text-center"
  >
    <button
      className="btn btn-danger"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      X
    </button>
  </td>
</tr>
`;
